universal-application-tool-0.0.1/app/services/program/ProgramServiceImpl.java

```diff
@@ -135,29 +135,34 @@ public ProgramDefinition getProgramDefinition(long id) throws ProgramNotFoundExc
 
   @Override
   @Transactional
-  public ProgramDefinition addBlockToProgram(long programId) throws ProgramNotFoundException {
```

Comment:
question: Perhaps a bit out of scope of this change, but why does this method exist if blocks must have a name/description?

Reply:
This method is used in the admin UI when we click the add block button. It adds a block with pre-populated fields without asking you to input. I guess this is for a smooth experience that users can edit the name/description later if needed.

Reply:
Got it, thank you!